frontend-react/scripts/cli/cli.ts

```diff
@@ -0,0 +1,211 @@
+import { Command } from "commander";
+import { preview, build } from "vite";
+import { getChildArgs, loadRedirectedEnv } from "./utils";
+import { spawnSync } from "node:child_process";
+import { exit } from "node:process";
+import { OUTPUT_PATH, getRegexes } from "./generateBrowserslistRegex";
+import { writeFileSync } from "node:fs";
+
+const program = new Command("reportstream-frontend-cli");
+program.option(
+    "--csp",
+    "use Content-Security-Policy headers in web server",
+    false,
+);
+program.option(
+    "-e, --env [ENVIRONMENT]",
+    "which special environment to configure for (ex: demo, trialfrontend, etc.)",
+);
+program.option("-d, --debug", "debug mode");
+program.option("--ci", "run in CI mode");
+
+const devCmd = program
+    .command("dev")
+    .description(
+        "run local dev server (pass vite-specific commands after '--')",
+    );
+devCmd.action((_, cmd: Command) => {
+    process.env.NODE_ENV = "development";
+    const opts = cmd.optsWithGlobals();
+    const env = loadRedirectedEnv(opts);
+    spawnSync("vite", ["dev", ...getChildArgs(process.argv)], {
+        env,
+        stdio: "inherit",
+    });
```

Comment:
can we make sure it still spins up at port `3000`? right now it's spinning up at: `localhost:5173`